internal/cache/cache_utils.go

```diff
@@ -331,6 +348,7 @@ type AviHTTPPolicyCache struct {
 	LastModified     string
 	InvalidData      bool
 	HasReference     bool
+	StringGroupRefs  []string
```

Comment:
Can there be multiple paths associated with one httppolicy ?

Reply:
Yes, all the paths for a hostname will be associated with single HttpPolicyset. One HttpRequestRule will be created for each path. And StringGroup will be created for each path in case useRegex is enabled.